tests/upgrades/test_performance_tuning.py

```diff
@@ -0,0 +1,153 @@
+"""Test for Performance Tuning related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from upgrade_tests import post_upgrade, pre_upgrade
+from robottelo import ssh
+from unittest2.case import TestCase
+
+
+MEDIUM_TUNING_DATA = "apache::mod::passenger::passenger_max_pool_size: 30" \
```

Comment:
Please use parentheses instead of backslash:
```python
MEDIUM_TUNING_DATA = ("apache::mod::passenger::passenger_max_pool_size: 30"
    "\napache::mod::passenger::passenger_max_request_queue_"
    ...
)
```

In this case, when you create data to save in file, instead of polluting strings with newline characters, it might be better idea to have a list of strings that will be joined before writing:
```python
MEDIUM_TUNING_DATA = ["apache::mod::passenger::passenger_max_pool_size: 30",
    "apache::mod::passenger::passenger_max_request_queue_"
    ...
]
with open('custom-hiera.yaml', 'w') as fd:
    fd.write("\n".join(DEFAULT_CUSTOM_HIERA_DATA))
```
`"".join()` inserts string between list elements, so you probably want to first merge both lists together.

Reply:
Fixed.

Reply:
Constants are lists now, but there are still strings with backslashes elsewhere. Please do not use backslash before newline, use parentheses instead.